workspace.json

```diff
@@ -10,12 +10,14 @@
     "@fluentui/ability-attributes": {
       "root": "packages/fluentui/ability-attributes",
       "projectType": "library",
-      "implicitDependencies": []
+      "implicitDependencies": [],
+      "tags": ["northstar"]
```

Comment:
we should be unify tags declaration. atm we use `vNext` for v9, `v8` for react https://github.com/microsoft/fluentui/pull/22604/files#diff-8615bf67663730c68237b51aa01c74d409e3ebfa0ddb23cde704c9a2e97779fd

if we wanna go with package name we should go with:
`react`
`react-components`
`react-northstar` 

althought the `react` tag for `v8` is missleading without additional context(docs)

thoughts?

Reply:
I don't mind, just pick a convention and go with it

Reply:
let's go with package names, it's much more realistic. I'll update to `react-norhtstar` in this PR and we can update the other packages in other PRs